adapters/nanointeractive/nanointeractive.go

```diff
@@ -0,0 +1,199 @@
+package nanointeractive
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+)
+
+type NanointeractiveAdapter struct {
```

Comment:
Nitpick: Is this the proper capitalization of the company or would `NanoInteractive` with a capital i be more appropriate?

Reply:
Thanks for this